tests/conformance_suite.postman_collection.json

```diff
@@ -31,18 +31,43 @@
 										],
 										"type": "text/javascript"
 									}
+								},
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											"const convertDidToEndpoint = (did) => {",
```

Comment:
If port is included in a did:web, the colon between host and port MUST be percent-encoded. This regex will need slight modification to account for that. The following is for reference, but I don't think it belongs here in a Postman request (will add more comments regarding this thought):

```javascript
const convertDidToEndpoint = (did) => {
  const re = /did:web:([a-zA-Z0-9.\-_%]+)(?::([a-zA-Z0-9.:\-_]+))?/;
  const match = did.match(re);
  if (!match) {
    throw new Error('DID is not a valid did:web');
  }
  const [host, path] = match.slice(1);
  const pathname = path?.split(':').join('/') || '.well-known';
  return `https://${decodeURIComponent(host)}/${pathname}/did.json`;
};
```

Note: would prefer not to encourage use of http here, even on localhost.

Reply:
I stole this from here: 
https://github.com/transmute-industries/verifiable-data/blob/main/packages/did-web/src/convertDidToEndpoint.ts

Reply:
This code will fail to convert a did:web DID with a port into a URL